client/src/styles/global.css

```diff
@@ -33,3 +33,8 @@
 /* NOTE: For the most part, allow React components to explicitely import as needed */
 @import url('./trumps/icon.css');
 @import url('./trumps/table.css');
+
+code > a,
+a:hover {
+  color: var(--global-color-accent--normal);
+}
```

Comment:
In #813, I propose this be placed in `./elements/text-semantics.css`.

<sub>I recognize the naming is not intuitive. It's a holdover from before I realized that. The stylesheet links to https://developer.mozilla.org/en-US/docs/Web/HTML/Element#Inline_text_semantics which lists `<a>` and `<code>`.</sub>